openhtf/io/frontend/src/app/utils.ts

```diff
@@ -41,6 +41,7 @@ export class SimplifyStatus extends MapPipe {
     'WAITING_FOR_TEST_START': 'WAITING',
     'STOP_WAIT': 'WAITING',
     'TIMEOUT': 'ERROR',
+    'ABORTED': 'ERROR',
```

Comment:
if you say so, but i don't see how this possibly works when the input status is "RUNNING' and there is no 'RUNNING' key in your map, is there no KeyError equivalent in ts?